src/content/peerconnection/audio/js/main.js

```diff
@@ -36,6 +36,9 @@ const offerOptions = {
   voiceActivityDetection: false
 };
 
+// Enabling opus DTX is an expert option without GUI.
+let useDtx = false;
```

Comment:
The linter is anal.
let useDtx; useDtx = false will presumably pass. Or find the magic "lint, lint go away" cantrip.

Reply:
we've run into that before in bandwidth.js ;-)